packages/engine/tests/simulations/mod.rs

```diff
@@ -0,0 +1,75 @@
+mod experiment;
+
+/// Opens `$project` relatively to this file as HASH project and uses
+/// `$project/integration-test.json` as configuration for the test.
+///
+/// The configuration contains a list of objects, where each object has the following values:
+/// - for a simple experiment:
+///   - "experiment": Name of the experiment to run as specified in `experiments.json`
+///   - "expected-outputs": List of expected outputs, where the length of the list must be equal to
+///     the number of simulations of the experiment. Each entry of the list is an objects with the
+///     step number as key mapping to the expected outputs of the corresponding step in the
+///     simulation run:
+///     - "json-state": subset of the values expected in the `json_state.json` output
```

Comment:
I think we need to better explain what a "subset" is somewhere. Maybe not necessarily in the documentation for the macro but we should explain you put fields you want to check the values of and all others are ignored.

Reply:
I updated the description and avoided the term "subset". It should now be more clear.